app/src/lib/git/diff-index.ts

```diff
@@ -82,7 +82,7 @@ export async function getIndexChanges(
   const args = ['diff-index', '--cached', '--name-status', '--no-renames', '-z']
 
   let result = await git(
-    [...args, 'HEAD', '--'],
```

Comment:
Not sure what happened to your merge, but this change is part of an important bugfix #2990 and some methods that were removed in #2991 are now restored.

Reply:
Merged the new changes correctly with my latest commit.